index.md

```diff
@@ -1,4 +1,125 @@
-# This is the **HOMEPAGE**.
-Refer to [Markdown](http://daringfireball.net/projects/markdown/) for how to write markdown files.
-## Quick Start Notes:
-1. Add images to the *images* folder if the file is referencing an image.
+---
+title: Welcome to Blazor Docs website!
+---
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>BLAZOR</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="#">Version Notes</a><span>|</span><a class="github-link" href="https://github.com/dotnet/docfx">View in Github</a>
+    </div>
+    <div class="minitext">
+        Blazor is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly
+    </div>
+    <div class="buttons-unit">
+      <a href="#" class="button"><i class="glyphicon glyphicon-send"></i>Get Started</a>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-wrench"></i>
+        <section>
+          <h2>Build Web UI with C#</h2>
+          <p class="lead">Blazor is a web UI framework, similar to existing JavaScript frameworks like Angular or React, but you write C# instead of JavaScript.</p>
```

Comment:
* Change "like" to "such as" for the examples.
* Since Angular and React are examples of "frameworks," use "and" between them.

> Blazor is a web UI framework, similar to existing JavaScript frameworks such as Angular and React, but you write C# instead of JavaScript.

Reply:
Okay